### ISSUE TITLE
Line replyCarouselTemplate 4th parameter isn't optional

### ISSUE THREAD

**Issue by gcobcqwe** [Yoctol/messaging-apis#278]
[replyCarouselTemplate(token, altText, carouselItems, options)](https://github.com/Yoctol/messaging-apis/tree/master/packages/messaging-api-line#replycarouseltemplatetoken-alttext-carouselitems-options)

The replyCarouselTemplate example need to input four parameters rather than three parameters.
Such as:
```javascript
client.replyCarouselTemplate(REPLY_TOKEN, 'this is a carousel template', [
  {
    thumbnailImageUrl: 'https://example.com/bot/images/item1.jpg',
    title: 'this is menu',
    text: 'description',
    actions: [
      {
        type: 'postback',
        label: 'Buy',
        data: 'action=buy&itemid=111',
      },
      {
        type: 'postback',
        label: 'Add to cart',
        data: 'action=add&itemid=111',
      },
      {
        type: 'uri',
        label: 'View detail',
        uri: 'http://example.com/page/111',
      },
    ],
  },
  {
    thumbnailImageUrl: 'https://example.com/bot/images/item2.jpg',
    title: 'this is menu',
    text: 'description',
    actions: [
      {
        type: 'postback',
        label: 'Buy',
        data: 'action=buy&itemid=222',
      },
      {
        type: 'postback',
        label: 'Add to cart',
        data: 'action=add&itemid=222',
      },
      {
        type: 'uri',
        label: 'View detail',
        uri: 'http://example.com/page/222',
      },
    ],
  },
],JSON.parse('{"imageAspectRatio":"square","imageSize":"contain"}'));

**Comment by tw0517tw** [comment]
Hi @gcobcqwe ,

The 4th parameter `option` mentioned in the document is optional, so it does not appear in the example code. It will use default value it you didn't pass it in.

ref: https://developers.line.me/en/docs/messaging-api/reference/#carousel

**Comment by gcobcqwe** [comment]
Sure,it's optional.
But the 4th parameter is not nullable value in replyCarouselTemplate function. 
Following the example , if the 4th parameter does not using ,it's will crash.
Maybe you can change the replyCarouselTemplate  4th parameter to nullable. Thanks.
[image](https://imgur.com/GXdH3ow.jpg)

**Comment by tw0517tw** [comment]
OK, that's definitely a bug and will be fixed.

**Comment by tw0517tw** [comment]
fixed in `messaging-api-line@v0.6.12`
